packages/verkle/src/verkleTree.ts

```diff
@@ -24,6 +21,8 @@ import {
   type VerkleTreeOptsWithDefaults,
 } from './types.js'
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import type { createVerkleTree } from './constructors.js' // Imported so intellisense can display docs
```

Comment:
Ah, yeah, I stumbled over this as well at some point. It would really be good if this can be solved on a more global scale (all such imports auto-added or so), this problem is all over the place and it would be good if we could use this `@link` feature from the docs more reliably.

(just as some PR-unrelated side note)

Reply:
Yup, have done some research on this but haven't quite solved it yet. We need to somehow link via something like `{@link module#referebcedType }` but I haven't figured out specifically how to implement yet.